src/api/error.js

```diff
@@ -22,7 +25,7 @@ const handleError = async ( e: Object, options: Object = {} ): Object => {
   if ( !e.response ) { throw e; }
   const errorText = await e.response.text( );
   const error = new INatApiError( errorText );
-  console.error(
+  logger.error(
```

Comment:
If `options.throw` is true we'll end up logging this as an exception... but maybe it is a good idea to log all failed requests for now. In a production context I imagine that might become excessive, though.

Reply:
Yes, my thoughts there were to increase logging because I could not reproduce the JWT errors on my devices. Hoping, that more logging will increase our chances of finding the problem here. I can leave a TODO to remove this in the future.